packages/css-reset/CssReset.md

```diff
@@ -1,7 +1,7 @@
 ```js static
 // App.jsx
 
-import '@tds/core-css-reset/dist/index.css'
+import '~@tds/core-css-reset/dist/index.css'
```

Comment:
Actually, the "~" is not needed here. Please remove.

This import statement is in a jsx file. The "~" is specific to Sass imports.

See starter kit here: https://github.com/telusdigital/telus-isomorphic-starter-kit/blob/master/ui/src/components/App/index.jsx#L4

Reply:
I see, I made this in response to another developer's use case who appeared to be adding css reset to their application's main SCSS file. I'll remove the tilde and rely on developer discretion.